engine/src/test/java/com/datatorrent/stram/cli/ApexCliTest.java

```diff
@@ -258,4 +258,26 @@ public void testMergeAppFromConfigAndAppPackage() throws Exception
 
     Assert.fail("Cli failed throw multiple apps exception.");
   }
+
+  @Test
+  public void testGetOperatorClassesCommandLineInfo() throws Exception
+  {
+    // The second and third arg values in the method call can be any strings.
+    // The unit test only checks that the corresponded values of the output parser options
+    // are equal to the same values that were passed in the method call.
+
+    String arg2 = "com.example.mydtapp.Application";
+    String arg3 = appFile.getAbsolutePath();
+
+    ApexCli.GetOperatorClassesCommandLineInfo commandLineOptions =
+        ApexCli.getGetOperatorClassesCommandLineInfo(new String[] {"-parent", arg2, arg3 });
+
+    Assert.assertEquals(commandLineOptions.parent, arg2);
+    Assert.assertEquals(commandLineOptions.args.length, 1);
+
+    if (commandLineOptions.args.length == 1) {
```

Comment:
Is the test necessary? The test will either fail on assert or commandLineOptions.args.length == 1.